config/schema_descriptions.yml

```diff
@@ -0,0 +1,72 @@
+reporting_patient_states:
+  description: "Monthly summary of a patient's information and health indicators. This table has one row per patient, per month, from the month of the patient's registration."
+  columns:
+    patient_id: "ID of the patient"
+    recorded_at: "Time (in UTC) at which the patient was registered"
+    status: "active, dead, migrated, etc"
+    gender: "Gender of the patient"
+    age: "Age of the patient as entered by a nurse"
+    age_updated_at: "Time (in UTC) at which the age was last updated"
+    date_of_birth: "Date of birth of the patient"
+    current_age: "Computed age as of today, given age, age-updated-at and date-of-birth. This will have the same value for a given patient across all rows."
```

Comment:
This one is a doozy. Hmm, how about something like:

> Patient's age as of today, based on `age`, `age-updated-at`, and `date-of-birth`. This will...

I'm also sensing that we have some restrictions on what characters we can use in these comments. Feel free to us `Age of the patient`. The confusion for me mostly stems from "given age".

Reply:
Yup, makes sense, I've updated it.